### PR TITLE
update installation document

### PR DESCRIPTION
**Pull request by tanhe123**

### CODE REVIEW

File: docs/usage/installation.md [hunk 1]
```diff
@@ -27,3 +59,9 @@ $ fun
     deploy              Deploy a project to AliCloud
     build               Build the dependencies
 ```
+
+## Install Fun by Downloading The Latest Binary
+
+Open the [Releases](https://github.com/aliyun/fun/releases) page, click a link of the corresponding platform to download the Fun zip package.
+
+After downloading, unzip it and use it directly.
```
Comment (vangie) [inline docs/usage/installation.md]: After downloaded,

File: docs/usage/installation.md [hunk 2]
```diff
@@ -1,6 +1,38 @@
 # Installation
 
-Fun is a command line tool developed for Node.js, however it also support Python, Java or other runtime enviroments. It can be installed by [npm](https://www.npmjs.com/):
+When using Fun, if you need to run and debug function locally, you need to use the fun local subcommand.
+
+Using fun local subcommand requires pre-installing docker. If you don't want to run and debug function locally, docker is not required.
```
Comment (vangie) [inline docs/usage/installation.md]: Fun local subcommand depend on docker.

File: docs/usage/installation.md [hunk 3]
```diff
@@ -1,6 +1,38 @@
...
+
+If you encounter network problems, you can use Alibaba Cloud Docker CE mirror to download. please refer to this [tutorial](https://yq.aliyun.com/articles/110806).
+
+### Configuring Docker Registry Mirror
+
+Once docker is installed, you can use docker to download the image. If you encounter network problems, it is recommended to configure aliyun [Registry Mirror](https://yq.aliyun.com/articles/29941).
+
+## Install Fun Using Npm
+
+Fun can be installed by [npm](https://www.npmjs.com/):
```
Comment (vangie) [inline docs/usage/installation.md]: could be

File: docs/usage/installation.md (hunk 1 above)
Comment (tanhe123) [reply docs/usage/installation.md]: https://english.stackexchange.com/questions/67576/after-downloaded-vs-after-downloading

File: docs/usage/installation.md (hunk 3 above)
Comment (tanhe123) [reply docs/usage/installation.md]: done

File: docs/usage/installation.md (hunk 2 above)
Comment (tanhe123) [reply docs/usage/installation.md]: done

**Pull request merged**